### DISCUSSION TITLE
Choose Simbrief FPL in the EFB or MCDU INIT page

### DISCUSSION THREAD

**Discussion by pacha35** [flybywiresim/aircraft#5582] [Help] [answered]
Hi, 
I have several FPL in Simbrief, but each time I run the EFB, it's always the last created Simbrief FPL which is choosen. It's the same thing with INIT REQUEST in the A32NX MCDU.
Is there a method to choose a FPL than in the World page?
Thanks

**Comment by tracernz** [comment, marked as answer]
SimBrief only allows you to have one OFP doesn't it?

**Reply by pacha35** [reply]
Yes, MCDU load only the last FPL created with SimBrief, we have not the possibility to choose another.

**Reply by tracernz** [reply]
> 
> 
> Yes, MCDU load only the last FPL created with SimBrief, we have not the possibility to choose another.

Forget about A32NX for a moment. You cannot generate more than one OFP on the SimBrief platform can you? If you generate a new one, it deletes your last OFP, so you can only ever have one.

**Reply by pacha35** [reply]
Hi, 
thanks for your answer, you are right about SimBrief. Then if I want to read another FPL, I have to change the date before to create the OFP.
Thank you again.

**Reply by tracernz** [reply]
> 
> 
> Hi,
> thanks for your answer, you are right about SimBrief. Then if I want to read another FPL, I have to change the date before to create the OFP.
> Thank you again.

Yeah, unfortunately I don't think it's possible for us to do much, as we only deal with OFPs (and that's what the SimBrief API provides to us, the single latest OFP).

**Reply by pacha35** [reply]
No problem, changing the date is a good solution.

**Comment by MrAstroApe** [comment]
> Hi, 
> 
> I have several FPL in Simbrief, but each time I run the EFB, it's always the last created Simbrief FPL which is choosen. It's the same thing with INIT REQUEST in the A32NX MCDU.
> 
> Is there a method to choose a FPL than in the World page?
> 
> Thanks

You can create many flight plans but you can only generate one plan. Generate a plan which you are going to fly.

**Comment by TheSledge-SLEJ** [comment]
I think he means the option to choose a COROUTE by name from pre-stored alternatives e.g. EGLLLOWI01 or EGKKLBNA01
